src/daemon.js

```diff
@@ -0,0 +1,98 @@
+const { info } = require('./log');
+
+const LEGACY_0_8_UPGRADE_DOC = '_design/medic:staged';
+const HORTI_UPGRADE_DOC = 'horti-upgrade';
+
+const Apps = require('./apps'),
+      DB = require('./dbs'),
```

Comment:
I think only classes should be capitalised. Can we change both of these to lowercase? `DB` feels like a special case but I'm not sure why...

Reply:
Yeah I don't know why Apps is capitalised either, it's just how it was from the start. DB is a acronym so it makes sense to be capitalised to me

Reply:
Clearly it should be `dB` then!